google-cloud-storage/support/doctest_helper.rb

```diff
@@ -1227,11 +1241,18 @@ def done_rewrite gapi
   Google::Apis::StorageV1::RewriteResponse.new done: true, resource: gapi
 end
 
-def random_bucket_hash(name = "my-bucket",
-  url_root="https://www.googleapis.com/storage/v1", location="US",
-  storage_class="STANDARD", versioning=nil, logging_bucket=nil,
-  logging_prefix=nil, website_main=nil, website_404=nil)
+def random_bucket_hash name: "my-bucket",
+                       url_root: "https://www.googleapis.com/storage/v1",
+                       location: "US",
+                       storage_class: "STANDARD",
+                       versioning: nil,
+                       logging_bucket: nil,
+                       logging_prefix: nil,
+                       website_main: nil,
+                       website_404: nil,
+                       public_access_prevention: "enforced"
```

Comment:
Is this supposed to be a really generic test fixture? If so I'd expect PAP to be unspecified (or not sent at all) to start.

Reply:
Updated, thanks!